test/e2e/layer.test.js

```diff
@@ -0,0 +1,47 @@
+const { assert, driver } = require('vl-ui-core').Test.Setup;
+const VlMapLayerPage = require('./pages/vl-map-layer.page');
+
+describe('vl-map-layer', async() => {
+    const page = new VlMapLayerPage(driver);
+
+    before(() => {
+        return page.load();
+    });
+
+    it('Als gebruiker kan ik de url en type van een grb baselayer bevragen', async () => {
+        const baseLayerGrb = await page.getBaseLayerGrb();
+
+        await assert.eventually.equal(baseLayerGrb.getUrl(), 'https://tile.informatievlaanderen.be/ws/raadpleegdiensten/wmts');
+        await assert.eventually.equal(baseLayerGrb.getType(), 'wmts');
+    });
```

Comment:
Misschien best verplaatsen vl-map, deze demo pagina focust op vl-map-layers.

Reply:
of splitsen in layer.test.js en baselayer.test.js